doc/proxy-clients.rst

```diff
@@ -184,6 +184,39 @@ dispatcher as explained above and pass it to the Symfony client::
 
     $symfony = new Symfony($httpDispatcher);
 
+
+The `HttpDispatcher` sends real HTTP requests using any instance of `HttpAsyncClient`
+available in your application (see HTTPlug_ for more information). If you
+have a single server Symfony application setup, you can send these requests
+directly to the cache kernel inside the same PHP process instead of sending actual
+HTTP requests over the network. This makes your setup easier as you don't need
+to know the IP of your server and will also save server resources.
+For that to work, you can use the `KernelDispatcher` instead of the `HttpDispatcher`.
```

Comment:
in rst. you need double backticks for code. single backtick is only for references.